### ISSUE TITLE
set status to "connect" only after finish fully connecting

### ISSUE THREAD

**Issue by ez1976** [netbirdio/netbird#2162]
Hello.
since we have a few policies and about 350 routes, it actually takes about 10-15 seconds for the client to fully be operational.

i think it would be wise if the netbird-ui icon will change to orange ONLY after all the routes are received, the peers are connected and the nameservers are available.

the better way is to have a different color or spinning icon while it is still connecting.

**Comment by hurricanehrndz** [comment]
I can submit a PR for this. I do need an svg of the netbird logo. I have custom fork that does this already

**Comment by hurricanehrndz** [comment]
DNS will be an issue, because not everyone applies internal DNS settings, in my fork I based full connected if a highly available source is resolvable, but that won't work for everyone. I can base it on DNS settings alone and if they have been applied to the system perhaps?

**Comment by hurricanehrndz** [comment]
I believe this is a Duplicate #2154

**Comment by gecube** [comment]
Yes, duplicate, but let's discuss what we really want to achieve.

**Comment by hurricanehrndz** [comment]
I am working on PR for this, what platform are you so that I ensure that is covered when I submit it

**Comment by ez1976** [comment]
Thanks.
I personally work on ubuntu but i have test machines on win10, win 11,
macos and fedora

**Comment by TechHutTV** [comment]
Tracked under #2154 — both threads describe the same UX gap during the route-settling phase (UI shows "connected" before the data plane is fully ready). Please subscribe there for updates. Closing as duplicate.
